extensions/amp-fx-scroll-sync/0.1/amp-fx-scroll-sync.js

```diff
@@ -55,6 +56,14 @@ class AmpScrollSync extends AMP.BaseElement {
       config['scale-origin'] = this.element
           .getAttribute('scale-origin');
       scrollSyncEffect = new ScrollSyncScaleEffect(element, config);
+    } else if (this.effectName_ == 'rotate') {
+      config['rotate-angle'] = getLengthNumeral(this.element
```

Comment:
Why are we parsing the config in the element instead of the specific implementation?